[Bug]: CWBI prod rating spec gateway time out in cwms-vue

### Describe the bug

when connecting to CWMS-Vue rating specs will not load.  getting the following error.



### Expected behavior

rating specs load correctly

### To Reproduce

request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/effective-dates?office-mask=LRL&rating-id-mask=*

log into cwms-vue to produciton CDA.  rating specs error out

### Priority

High

### Logs/Incident Identifier

14:24:55 SEVERE: Error eager loading rating specs from database
hec.db.DbIoException: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/effective-dates?office-mask=LRL&rating-id-mask=*
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at wcds.dbi.cwms.data.api.CwmsDaoImpl.wrapIOException(CwmsDaoImpl.java:77)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveRatingDates(CwmsRatingCdaDao.java:206)
        at hec.cwmsVue.CwmsRatingPanelFX.retrieveSpecDates(CwmsRatingPanelFX.java:229)
        at hec.cwmsVue.CwmsRatingPanelFX.lambda$loadRatingCatalog$3(CwmsRatingPanelFX.java:280)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/effective-dates?office-mask=LRL&rating-id-mask=*
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.checkError(HttpRequestBuilderImpl.java:278)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.handleExecutionError(HttpRequestBuilderImpl.java:264)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.execute(HttpRequestBuilderImpl.java:247)
        at mil.army.usace.hec.cwms.data.api.client.controllers.RatingController.retrieveRatingEffectiveDates(RatingController.java:66)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveRatingDates(CwmsRatingCdaDao.java:201)
        ... 6 more

14:24:55 SEVERE: Error eager loading rating specs from database
hec.db.DbIoException: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/effective-dates?office-mask=LRL&rating-id-mask=*
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at wcds.dbi.cwms.data.api.CwmsDaoImpl.wrapIOException(CwmsDaoImpl.java:77)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveRatingDates(CwmsRatingCdaDao.java:206)
        at hec.cwmsVue.CwmsRatingPanelFX.retrieveSpecDates(CwmsRatingPanelFX.java:229)
        at hec.cwmsVue.CwmsRatingPanelFX.lambda$loadRatingCatalog$3(CwmsRatingPanelFX.java:280)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/effective-dates?office-mask=LRL&rating-id-mask=*
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.checkError(HttpRequestBuilderImpl.java:278)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.handleExecutionError(HttpRequestBuilderImpl.java:264)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.execute(HttpRequestBuilderImpl.java:247)
        at mil.army.usace.hec.cwms.data.api.client.controllers.RatingController.retrieveRatingEffectiveDates(RatingController.java:66)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveRatingDates(CwmsRatingCdaDao.java:201)
        ... 6 more

14:26:08 SEVERE: Error lazy loading rating specs from database
hec.db.DbIoException: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/metadata?page-size=1000&timezone=UTC&office=LRL
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at wcds.dbi.cwms.data.api.CwmsDaoImpl.wrapIOException(CwmsDaoImpl.java:77)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveFullRatingCatalog(CwmsRatingCdaDao.java:562)
        at hec.cwmsVue.CwmsRatingPanelFX.retrieveRatingCatalog(CwmsRatingPanelFX.java:128)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/metadata?page-size=1000&timezone=UTC&office=LRL
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.checkError(HttpRequestBuilderImpl.java:278)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.handleExecutionError(HttpRequestBuilderImpl.java:264)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.execute(HttpRequestBuilderImpl.java:247)
        at mil.army.usace.hec.cwms.data.api.client.controllers.RatingController.retrieveRatingMetadata(RatingController.java:105)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveFullRatingCatalog(CwmsRatingCdaDao.java:547)
        ... 5 more

14:26:08 SEVERE: Error lazy loading rating specs from database
hec.db.DbIoException: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/metadata?page-size=1000&timezone=UTC&office=LRL
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at wcds.dbi.cwms.data.api.CwmsDaoImpl.wrapIOException(CwmsDaoImpl.java:77)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveFullRatingCatalog(CwmsRatingCdaDao.java:562)
        at hec.cwmsVue.CwmsRatingPanelFX.retrieveRatingCatalog(CwmsRatingPanelFX.java:128)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: mil.army.usace.hec.cwms.http.client.CwmsHttpResponseException: Unknown error occurred for request: GET https://cwms-data.usace.army.mil/cwms-data/ratings/metadata?page-size=1000&timezone=UTC&office=LRL
Error code: 504 Gateway Time-out
<html>
<head><title>504 Gateway Time-out</title></head>
<body>
<center><h1>504 Gateway Time-out</h1></center>
</body>
</html>

        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.checkError(HttpRequestBuilderImpl.java:278)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.handleExecutionError(HttpRequestBuilderImpl.java:264)
        at mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl$HttpRequestExecutorImpl.execute(HttpRequestBuilderImpl.java:247)
        at mil.army.usace.hec.cwms.data.api.client.controllers.RatingController.retrieveRatingMetadata(RatingController.java:105)
        at wcds.dbi.cwms.data.api.CwmsRatingCdaDao.retrieveFullRatingCatalog(CwmsRatingCdaDao.java:547)
        ... 5 more

### CURL Commands

```shell

```

### CDA Version

 2026.05.12-f

### Additional context

_No response_

--- a/cwms-data-api/build.gradle
+++ b/cwms-data-api/build.gradle
@@ -282,6 +282,7 @@ task run(type: JavaExec) {
     jvmArgs += "-DwarContext=/" + context
     jvmArgs += "-Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true"
     jvmArgs += "-DAUTH_RE_ENABLE_NON_HASH_KEY_SUPPORT=true"
+    jvmArgs += "-Duser.timezone=UTC"
     // If you have the docker-compose environment up and are trying to run
     // CDA from run to debug uncomment the following lines.
     //jvmArgs += "-Dcwms.dataapi.access.providers=KeyAccessManager,CwmsAccessManager,OpenID"

